src/views/verify.tsx

```diff
@@ -1,9 +1,86 @@
 import { isValid, openAttestationVerifiers, verificationBuilder, VerificationFragment } from "@govtechsg/oa-verify";
+import { providers } from "ethers";
 import React, { useEffect, useState } from "react";
 import { FailedAlert, SucceedAlert } from "../components/alert";
 import { Status } from "../shared";
 
-type Network = "ropsten" | "homestead" | "rinkeby" | "goerli";
+enum Network {
+  mainnet = 1,
+  ropsten = 3,
+  rinkeby = 4,
+  goerli = 5,
+  matic = 137,
+  maticmum = 80001,
+}
```

Comment:
umm... Not sure if this is still necessary because it seems like your `supportedNetworks` below already have the ids?

Reply:
ok